src/game/gamecore.cpp

```diff
@@ -388,7 +389,15 @@ void CCharacterCore::Move()
 		}
 	}
 
-	m_Pos = NewPos;
+	// check player death by deathtiles
+	vec2 LastDeathPos;
+	if(m_pCollision->HitTileDeath(m_Pos, NewPos, LastDeathPos, 28.0f/2))//Gets tee radius
```

Comment:
Indent issues around here

Reply:
I can't find any oO pls explain

Reply:
![image](https://user-images.githubusercontent.com/355114/56895387-49a31880-6a89-11e9-9722-b33bb70f6d46.png)

Probably a mix of spaces and tabs

Reply:
probably a problem from my IDE, i will take a look

Reply:
is the intended way tabs or spaces?

Reply:
Tabs

Reply:
fixed